sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6296,4 +6296,19 @@ ALTER TABLE exportconfiguration_history
 
 INSERT INTO schema_version (version_number, comment) VALUES (313, 'Allow specific users to create public custom exports #1754');
 
+--2020-02-02
+ALTER TABLE exposures RENAME patientexpositionrole TO exposureRole;
+ALTER TABLE exposures_history RENAME patientexpositionrole TO exposureRole;
+
+UPDATE exposures SET exposureRole = NULL WHERE exposureRole = 'NOT_COLLECTED';
+UPDATE exposures SET exposureRole = 'STAFF' WHERE exposureRole = 'WORKING_AT';
+UPDATE exposures SET exposureRole = 'GUEST' WHERE exposureRole = 'ACCOMMODATED_IN';
+UPDATE exposures SET exposureRole = 'PATIENT' WHERE exposureRole = 'CARED_FOR';
+
+UPDATE exposures_history SET exposureRole = NULL WHERE exposureRole = 'NOT_COLLECTED';
```

Comment:
It's not necessary to change the values in the history tables (actually it's probably a bad idea because it doesn't allow backtracking what the original values were in case someone is unhappy with the migration)

Reply:
updated